docs/index.rst

```diff
@@ -71,22 +71,19 @@ This tool helps transform legacy configuration files templated with the atparse
 
 | :any:`CLI documentation with examples<cli_template_translate_examples>`
 
-File Provisioning
-^^^^^^^^^^^^^^^^^
+File/Directory Provisioning
```

Comment:
Do we want this to mirror the description in file.py? Maybe make both `File and directory staging` or `File/Directory Provisioning`?

Reply:
I'll push an update to `api/file.py`. If you had something else in mind, please let me know. It looks like the other modules called `file.py` are updated, but please let me know if I missed something.

Reply:
Oh thats fine, I was pondering naming out loud, that works for me!